bindgen/src/spec.ts

```diff
@@ -68,6 +69,15 @@ const schemaFile = new URL("../generated/spec.schema.json", import.meta.url);
 const schemaJson = JSON.parse(fs.readFileSync(schemaFile, { encoding: "utf8" }));
 export const validate = ajv.compile<RelaxedSpec>(schemaJson);
 
+const optInSchemaFile = new URL("../generated/opt-in-spec.schema.json", import.meta.url);
```

Comment:
As discussed in person, I'd rather we not do this, or at least not do it like this. Since we can only have one schema for each file, I'd prefer that we use the binding generator to generate this file from the spec.yaml (and the sdk-specific specs) and listing all known classes/records and their methods/fields. This would enable some nice autocomplete while editing the opt-in list. This seems much more valuable to me than the trivial structural validation that we get for such a simple yaml file.

Reply:
From our discussion, yes I agree! Having the autocomplete will be very helpful when working with the spec. I'll revert the initial validation 👍